packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/flush-archive/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "pci_projects_project_storages_containers_container_cold_archive_flush_title": "Voulez-vous vider votre archive ?",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_description": "Vous êtes sur le point de supprimer le contenu de l'archive : {{ containerName }}.",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_terminate_input_label": "Entrez \"TERMINATE\" dans le champ ci-dessous pour confirmer",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_submit_label": "Confirmer",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_cancel_label": "Annuler",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_archive_erase_message": "Toutes les données sur le conteneur d'archive {{containerName}} seront perdues.",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_archive_status_archived_restored_warning": "Votre archive {{containerName}} contient {{objetsCount}} objet(s).</br>Vous êtes sur le point de vider l'archive de tout son contenu.",
```

Comment:
**nitpick:** Split into 2 sentences and avoid `,br>` tag in translation files.

Reply:
Done.